src/main/java/com/courtcanva/ccfranchise/CcFranchiseApplication.java

```diff
@@ -2,8 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 @SpringBootApplication
+@EnableJpaRepositories
+@EnableJpaAuditing
```

Comment:
why we need this?

Reply:
remove @EnableJpaRepositories
@EnableJpaAuditing  for @CreationTimestamp and @UpdateTimestamp